docs/src/modules/components/AdCarbon.js

```diff
@@ -1,14 +1,32 @@
 import * as React from 'react';
-import GlobalStyles from '@mui/material/GlobalStyles';
+import { styled } from '@mui/material/styles';
 import loadScript from 'docs/src/modules/utils/loadScript';
 import AdDisplay from 'docs/src/modules/components/AdDisplay';
 import { adStylesObject } from 'docs/src/modules/components/ad.styles';
 
+const CarbonRoot = styled('span')(({ theme }) => {
+  const styles = adStylesObject['body-image'](theme);
+
+  return {
+    '& #carbonads': {
+      ...styles.root,
+      '& .carbon-img': styles.imgWrapper,
+      '& img': styles.img,
+      '& a, & a:hover': styles.a,
+      '& .carbon-text': styles.description,
+      '& .carbon-poweredby': styles.poweredby,
+    },
+    '& [id^=carbonads_]': {
+      display: 'none',
+    },
```

Comment:
```suggestion
    '&>div': {
      display: 'none',
    },
```

Maybe more solid since it does not rely on how carbon manages its multiple ads

Reply:
Could also be based on 

```css
&>div: { display: 'hidden' },
&>div:last-of-type: { ... }
```

Reply:
I have changed the approach, this should be more resilient, inspired by your comment.